script/staging/FullStagingDeploy.s.sol

```diff
@@ -328,6 +291,22 @@ contract StagingDeploymentBase is Script {
             mintRecipients: mintRecipients
         });
 
+        // Extra rate limit configuration
+        MainnetControllerInit.setRateLimitData({
+            key:        MainnetController(instance.controller).LIMIT_USDE_MINT(),
+            rateLimits: IRateLimits(instance.rateLimits),
+            data:       rateLimitData6,
+            name:       "usdeMintData",
+            decimals:   6
+        });
+        MainnetControllerInit.setRateLimitData({
+            key:        MainnetController(instance.controller).LIMIT_USDE_BURN(),
+            rateLimits: IRateLimits(instance.rateLimits),
+            data:       rateLimitData18,
+            name:       "usdeBurnData",
+            decimals:   18
+        });
```

Comment:
Need to add 4626 deposit/withdraw rate limits for susde

Reply:
ah yes.. ill manually add to current deploy as well